zndraw/app.py

```diff
@@ -140,12 +140,19 @@ def update_cache(self):
         self.app.config["TUTORIAL"] = self.tutorial
         self.app.config["AUTH_TOKEN"] = self.auth_token
         self.app.config["USE_TOKEN"] = self.use_token
+        self.app.config["PORT"] = self.port
 
         self.app.config["upgrade_insecure_requests"] = self.upgrade_insecure_requests
         self.app.config["compute_bonds"] = self.compute_bonds
 
         setup_cache()
         cache.set("FILEIO", self.fileio)
+        # Mapping of modifier name f"name" to [request.sid, ...]
+        cache.set("ROOM_MODIFIER_HOSTS", {})
+        cache.set("MODIFIER_HOSTS", {})
+        # Mapping of modifier name f"name" to [dict, ...]
+        cache.set("ROOM_MODIFIER_SCHEMA", {})
```

Comment:
This might be a too large change, but I think we should remove `ROOM_MODIFIER_HOSTS`. Allowing people to register their own modifiers invites all sorts of trouble with timeouts. Now especially because running a modifier consumes a `celery` worker, which we don't get back until the modifier somehow finishes.

Reply:
Then we could have `DEFAULT_MODIFIERS` and `CUSTOM_MODIFIERS`, which ideally are sent to different queues.